configure.ac

```diff
@@ -3841,6 +3843,14 @@ then
 fi
 
 
+# Default optimization enable
```

Comment:
optimized is too generic especially without an explanation.  Pretend you're a user seeing this for the first time.  Does optimized mean optimized TLS, crypto, hardware, async, assembly, buffers, memory.. etc.  Preprocessor settings are not in the top several for me.